plugins/EpadDB.js

```diff
@@ -2414,46 +2416,68 @@ async function epaddb(fastify, options, done) {
-                        .on('close', () => {
-                          fastify.log.info(`${inputfolder}/dicoms.zip extracted`);
-                          fs.remove(`${inputfolder}/dicoms.zip`, (error) => {
-                            if (error) {
-                              fastify.log.info(
-                                `${inputfolder}/dicoms.zip file deletion error ${error.message}`
-                              );
-                              reject(error);
-                            } else {
-                              fastify.log.info(`${inputfolder}/dicoms.zip deleted`);
-                            }
-                          });
-                        })
-                        .on('error', (error) => {
-                          reject(
-                            new InternalError(`Extracting zip ${inputfolder}dicoms.zip`, error)
-                          );
-                        });
+                      // fs.createReadStream(`${inputfolder}/dicoms.zip`)
+                      //   .pipe(unzip.Extract({ path: `${inputfolder}` }))
+                      //   .on('close', () => {
+                      //     fastify.log.info(`${inputfolder}/dicoms.zip extracted`);
+                      //     fs.remove(`${inputfolder}/dicoms.zip`, (error) => {
+                      //       if (error) {
+                      //         fastify.log.info(
+                      //           `${inputfolder}/dicoms.zip file deletion error ${error.message}`
+                      //         );
+                      //         reject(error);
+                      //       } else {
+                      //         fastify.log.info(`${inputfolder}/dicoms.zip deleted`);
+                      //       }
+                      //     });
+                      //   })
+                      //   .on('error', (error) => {
+                      //     reject(
+                      //       new InternalError(`Extracting zip ${inputfolder}dicoms.zip`, error)
+                      //     );
+                      //   });
                       // un zip part over
                     });
+                  fastify.log.info(
+                    `calling prep download for project level files/folders for { project: projectid } : ${projectid} - {project_id: projectdbid} : ${projectdbid}`
+                  );
                   // eslint-disable-next-line no-await-in-loop
-                  await fastify.prepProjectDownload(
+                  const dicomPath = await fastify.prepProjectDownload(
                     request.headers.origin,
                     { project: projectid },
-                    { format: 'stream', includeAims: 'true' },
+                    { format: 'stream', includeAims: 'false' },
                     request.epadAuth,
                     writeStream,
                     {
                       project_id: projectdbid,
-                    }
+                    },
+                    undefined,
```

Comment:
you should remove these two undefineds. this is the new function signature (reqOrigin, params, query, epadAuth, output, whereJSON, returnFolder)